common/app/conf/switches.scala

```diff
@@ -830,6 +830,15 @@ object Switches {
     exposeClientSide = false
   )
 
+  val IdentityUseAvatarApi = Switch(
+    "Identity",
+    "id-use-avatar-api",
+    "If switched on, avatars will be uploaded using the new Avatar API",
+    safeState = Off,
+    sellByDate = new LocalDate(2015, 7, 1),
```

Comment:
Ah - from previous launch ideas. Will update for 30 days time (when Google switch off the GAE app this PR is replacing).